src/components/RootNavigator/NotificationCenterNavigator.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import {
+  createMaterialTopTabNavigator,
+  MaterialTopTabBarProps,
+} from "@react-navigation/material-top-tabs";
+import { useTranslation } from "react-i18next";
+import { useAnnouncements } from "@ledgerhq/live-common/lib/notifications/AnnouncementProvider";
+import { ScreenName } from "../../const";
+import NotificationCenterNews from "../../screens/NotificationCenter/News";
+import NotificationCenterStatus from "../../screens/NotificationCenter/Status";
+import { NavigationToggle } from "@ledgerhq/native-ui/components/Navigation/ToggleGroup/NavigationToggleGroup";
+import { ToggleGroupContainer } from "@ledgerhq/native-ui/components/Navigation/ToggleGroup/TemplateToggleGroup";
+import { Flex } from "@ledgerhq/native-ui";
+import styled from "styled-components/native";
+
+const Tab = createMaterialTopTabNavigator();
+
+const TabBarContainer = styled(Flex)`
+  border-bottom-width: 1px;
+  border-bottom-color: ${p => p.theme.colors.palette.neutral.c40};
+`;
+
+function TabBar({ state, descriptors, navigation }: MaterialTopTabBarProps) {
+  return (
+    <TabBarContainer paddingLeft={4} paddingRight={4} paddingBottom={4}>
+      <ToggleGroupContainer>
+        {state.routes.map((route, index) => {
+          const { options } = descriptors[route.key];
+          const label = options.title;
+
+          const isActive = state.index === index;
+
+          const onPress = () => {
+            const event = navigation.emit({
+              type: "tabPress",
+              target: route.key,
+              canPreventDefault: true,
+            });
+
+            if (!isActive && !event.defaultPrevented) {
+              navigation.navigate(route.name);
+            }
+          };
+
+          return (
+            <NavigationToggle
```

Comment:
Can you double check if the `key` prop is needed here?

Reply:
Nice find, didn't had warning but it can't hurt, added